deploy/docs/Troubleshoot_Collection.md

```diff
@@ -262,6 +262,41 @@ kubectl port-forward collection-sumologic-xxxxxxxxx-xxxxx 8080:24231
 
 Then, in your browser, go to `http://localhost:8080/metrics`. You should see Prometheus metrics exposed.
 
+#### Check the `/metrics` endpoint for Kubernetes services
+
+For kubernetes services you can use the following way:
+
+1. Create `sumologic-debug` pod
+
+   ```bash
+   cat << EOF | kubectl apply -f -
+   apiVersion: v1
+   kind: Pod
+   metadata:
+     name: sumologic-debug
+     namespace: <namespace you want to create pod in (e.g. sumologic)>
+   spec:
+     containers:
+     - args:
+       - receiver-mock
+       image: sumologic/kubernetes-tools:2.9.0
+       imagePullPolicy: IfNotPresent
+       name: debug
+     serviceAccountName: <service account name used by prometheus (e.g. collection-kube-prometheus-prometheus)>
+   EOF
+   ```
+
+2. Go into the container:
+
+   ```bash
+   kubectl exec -it sumologic-debug -n <namespace> bash
+
+3. Talk with API directly like prometheus does, e.g.
```

Comment:
I don't like "talk" here but it could stay :P

Reply:
Is it better now?